cv/CV-guide.md

```diff
@@ -0,0 +1,171 @@
+# Examples of cv term use for QC
+
+## Using existing term definitions to write a 
+### single value metric:
+If you would want to report the number of MS1 spectra for a particular run, you can use the cv Term
+```
+[Term]
+id: QC:4000111
+name: Number of MS1 spectra
+def: "Number of MS1 spectra" [PSI:QC]
+comment: A lower number of MS1 spectra acquired during one sample run compared to similar runs can indicate mismatched instrument settings or issues with the instrumentation.
```

Comment:
or issues with sample amount ...

Reply:
As soon as the term is fixed in the obo, this'll get an update